src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -1,249 +1,194 @@
-                 for="email">Email</label>
-          <div class="cd-col-form-input">
-            <input class="form-control"
-                   type="email"
-                   placeholder="Email..."
-                   id="email"
-                   name="email"
-                   formControlName="email">
-
-            <span class="invalid-feedback"
-                  *ngIf="userForm.showError('email', formDir, 'email')"
-                  i18n>Invalid email.</span>
-          </div>
-        </div>
-
-        <!-- Roles -->
-        <div class="form-group row">
-          <label class="cd-col-form-label"
-                 i18n>Roles</label>
-          <div class="cd-col-form-input">
-            <span class="no-border full-height"
-                  *ngIf="allRoles">
-              <cd-select-badges [data]="userForm.controls.roles.value"
-                                [options]="allRoles"
-                                [messages]="messages"></cd-select-badges>
-            </span>
-          </div>
-        </div>
-
-        <!-- Enabled -->
-        <div class="form-group row"
-             *ngIf="!isCurrentUser()">
-          <div class="cd-col-form-offset">
-            <div class="custom-control custom-checkbox">
-              <input type="checkbox"
-                     class="custom-control-input"
-                     id="enabled"
-                     name="enabled"
-                     formControlName="enabled">
-              <label class="custom-control-label"
-                     for="enabled"
-                     i18n>Enabled</label>
-            </div>
-          </div>
-        </div>
-
-        <!-- Force change password -->
-        <div class="form-group row"
-             *ngIf="!isCurrentUser() && !authStorageService.isSSO()">
-          <div class="cd-col-form-offset">
-            <div class="custom-control custom-checkbox">
-              <input type="checkbox"
-                     class="custom-control-input"
-                     id="pwdUpdateRequired"
-                     name="pwdUpdateRequired"
-                     formControlName="pwdUpdateRequired">
-              <label class="custom-control-label"
-                     for="pwdUpdateRequired"
-                     i18n>User must change password at next logon</label>
-            </div>
-          </div>
-        </div>
+           class="form-header">{{ action | titlecase }} {{ resource | upperFirst }}
+      </div>
+      <!-- UserName -->
+      <div class="form-item">
+        <cds-text-label for="username"
+                        cdRequiredField="Username"
+                        [invalid]="!userForm.controls.username.valid && userForm.controls.username.dirty"
+                        [invalidText]="usernameError"
+                        i18n>Username
+          <input cdsText
+                 placeholder="Username..."
+                 i18n-placeholder
+                 id="username"
+                 name="username"
```

Comment:
```suggestion
```

Reply:
changes are done

Reply:
We dont need to set `name` if we are having `formControlName` . Sorry I was not clear earlier in above comment. Thanks